src/clients/inst_scc.rb

```diff
@@ -304,12 +306,14 @@ def start_workflow
           next:  "register_addons"
         },
         "register_addons"        => {
-          abort: :abort,
-          next:  "update_autoyast_config"
+          abort:      :abort,
+          extensions: "select_addons",
+          next:       "update_autoyast_config"
         },
         "reregister_addons"      => {
-          abort: :abort,
-          next:  "check"
+          abort:      :abort,
+          extensions: "select_addons",
```

Comment:
Are you sure about this one? I have actually drawn the dialog transition graph for my feature, and this edge would continue to a different branch. Did you mean "select_addons_rereg" here?